documentation/modules/auxiliary/admin/ldap/vmware_vcenter_vmdir_auth_bypass.md

```diff
@@ -0,0 +1,119 @@
+## Vulnerable Application
+
+### Description
+
+This module bypasses LDAP authentication in VMware vCenter Server's
+vmdir service to add an arbitrary administrator user. Version 6.7
+prior to the 6.7U3f update is vulnerable.
+
+### Setup
+
+Tested in the wild. No setup notes available at this time.
+
+## Verification Steps
+
+Follow [Setup](#setup) and [Scenarios](#scenarios).
+
+## Actions
+
+### Add
+
+Add an admin user to the target.
```

Comment:
I would just clarify this and say target vCenter Server for clarity. Otherwise good though.